src/DynamoUtilities/AssemblyConfiguration.cs

```diff
@@ -13,7 +13,14 @@ public sealed class AssemblyConfiguration
 
         public static AssemblyConfiguration Instance
         {
-            get { return instance ?? new AssemblyConfiguration(); }
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new AssemblyConfiguration();
+                }
+                return instance;
+            }
```

Comment:
Was this a bug before (`instance` not being assigned `AssemblyConfiguration()` if `null`)?

Reply:
it was generating a warning.